File: go/8/main.go
package main

import (
    "fmt"
)

func somarDois(a float64, b float64) float64 {
    return a + b
}

func somarTres(a float64, b float64, c float64) float64 {
    return a + b + c
}

func main() {
    resultado1 := somarDois(5.0, 3.0)
    fmt.Println("Soma de 5.0 e 3.0:", resultado1)
	
    resultado2 := somarTres(5.0, 3.0, 2.0)
    fmt.Println("Soma de 5.0, 3.0 e 2.0:", resultado2)
}

File: python/8/main.py
from calculadora import *
if __name__ == "__main__":
    calc = Calculadora()
    
    resultado1 = calc.somar(5.0, 3.0)
    print("Soma de 5.0 e 3.0:", resultado1)
    
    resultado2 = calc.somar_três(5.0, 3.0, 2.0)
    print("Soma de 5.0, 3.0 e 2.0:", resultado2)